packages/accordion/src/AccordionContent.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from 'react'
```

Comment:
Can we rename this file to match the name of the component?  `AccordionContent` => `AccordionDetails`